pfrl/experiments/train_agent.py

```diff
@@ -53,14 +57,20 @@ def train_agent(
         while t < steps:
 
             # a_t
-            action = agent.act(obs)
```

Comment:
I'm just curious. Why do you have to modify `train_agent` even though you have `train_hrl_agent`?

Reply:
oh - this was for the test ddpg agent. Should I delete it?